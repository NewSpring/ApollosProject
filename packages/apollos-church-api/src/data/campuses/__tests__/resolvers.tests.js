import { graphql } from 'graphql';
import { createTestHelpers } from '@apollosproject/server-core/lib/testUtils';
import { campusSchema, peopleSchema } from '@apollosproject/data-schema';
import * as Campus from '../index';

const { getSchema, getContext } = createTestHelpers({ Campus });

describe('Campus', () => {
  const context = getContext();
  const schema = getSchema([campusSchema, peopleSchema]);
  it('gets campuses', async () => {
    const query = `
      query {
        campuses {
          image {
            uri
          }
        }
      }
    `;
    const result = await graphql(schema, query, {}, context);
    expect(result).toMatchSnapshot();
  });
});
