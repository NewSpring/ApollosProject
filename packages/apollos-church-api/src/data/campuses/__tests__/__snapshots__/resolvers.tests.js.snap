// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Campus gets campuses 1`] = `
Object {
  "data": Object {
    "campuses": null,
  },
  "errors": Array [
    [GraphQLError: Cannot read property 'value' of undefined],
  ],
}
`;
